src/plugins/intel_gpu/src/plugin/transformations/move_fc_reshape_to_weights.cpp

```diff
@@ -44,7 +44,8 @@ MoveFCReshapeToWeights::MoveFCReshapeToWeights() {
     auto weights_input_m = std::make_shared<ov::pass::pattern::op::Or>(ov::OutputVector{reshape_m, transpose_m});
 
     auto data_m = any_input();
-    auto fully_connected_m = wrap_type<op::FullyConnected>({data_m, weights_input_m});
+    auto bias_m = any_input();
+    auto fully_connected_m = wrap_type<op::FullyConnected>({data_m, weights_input_m, bias_m});
```

Comment:
nit: I think there's no big sense to have named variable for bias in such cases. Something like this should be shorter and won't lose any meaningful info
```
auto fully_connected_m = wrap_type<op::FullyConnected>({data_m, weights_input_m, any_input()});
```